### PR TITLE
Add ChannelFactory companion object to all channels

### PR DESCRIPTION
**Pull request by veptechno**
Now when passing in BotEngine some channels (Alice, Viber, Google) need to specify .Factory(), which is different from other channels and causes confusion.
This PR adds companion objects implemented by JaicpChannelFactory to these channels and allows you to pass channels like other

### CODE REVIEW

**Review by Denire — APPROVED** [review]

File: channels/google-actions/src/main/kotlin/com/justai/jaicf/channel/googleactions/jaicp/ActionsFulfillmentDialogflow.kt [hunk 1]
```diff
@@ -9,4 +9,9 @@ class ActionsFulfillmentDialogflow(
 ) : JaicpCompatibleChannelFactory {
     override val channelType = "dialogflow"
     override fun create(botApi: BotApi) = ActionsFulfillment.dialogflow(botApi, useDataStorage)
+
+    companion object : JaicpCompatibleChannelFactory {
```
Comment (nikvoloshin) [inline channels/google-actions/src/main/kotlin/com/justai/jaicf/channel/googleactions/jaicp/ActionsFulfillmentDialogflow.kt:13]: Let's use [delegation](https://kotlinlang.org/docs/delegation.html) for easier inheritance
```suggestion
    companion object : JaicpCompatibleChannelFactory by ActionsFulfillmentDialogflow()
```

File: channels/google-actions/src/main/kotlin/com/justai/jaicf/channel/googleactions/jaicp/ActionsFulfillmentSDK.kt [hunk 2]
```diff
@@ -9,4 +9,9 @@ class ActionsFulfillmentSDK(
 ) : JaicpCompatibleChannelFactory {
     override val channelType = "google"
     override fun create(botApi: BotApi) = ActionsFulfillment.sdk(botApi, useDataStorage)
+
+    companion object : JaicpCompatibleChannelFactory {
```
Comment (nikvoloshin) [inline channels/google-actions/src/main/kotlin/com/justai/jaicf/channel/googleactions/jaicp/ActionsFulfillmentSDK.kt:13]: The same about delegation

File: channels/viber/src/main/kotlin/com/justai/jaicf/channel/viber/ViberChannel.kt [hunk 3]
```diff
@@ -121,8 +121,12 @@ class ViberChannel private constructor(
         }
     }
 
-    companion object {
+    companion object : JaicpCompatibleAsyncChannelFactory {
```
Comment (nikvoloshin) [inline channels/viber/src/main/kotlin/com/justai/jaicf/channel/viber/ViberChannel.kt:124]: The same about delegation

File: channels/yandex-alice/src/main/kotlin/com/justai/jaicf/channel/yandexalice/AliceChannel.kt [hunk 4]
```diff
@@ -55,4 +51,11 @@ class AliceChannel(
         override val channelType = "yandex"
         override fun create(botApi: BotApi) = AliceChannel(botApi, oauthToken = "", useDataStorage = useDataStorage)
     }
+
+    companion object : JaicpCompatibleChannelFactory {
```
Comment (nikvoloshin) [inline channels/yandex-alice/src/main/kotlin/com/justai/jaicf/channel/yandexalice/AliceChannel.kt:55]: The same about delegation

**Review by nikvoloshin — CHANGES_REQUESTED** [review]

**Review by nikvoloshin — APPROVED** [review]

**Pull request merged**